tests/migrations/test_autodetector.py

```diff
@@ -3798,16 +3798,16 @@ def test_many_to_many_changed_to_concrete_field(self):
         # Right number/type of migrations?
         self.assertNumberMigrations(changes, "testapp", 1)
         self.assertOperationTypes(
-            changes, "testapp", 0, ["RemoveField", "AddField", "DeleteModel"]
+            changes, "testapp", 0, ["RemoveField", "DeleteModel", "AddField"]
```

Comment:
Unrelated changed, explained in https://code.djangoproject.com/ticket/31788#comment:5